src/test/java/org/folio/rest/api/TestBase.java

```diff
@@ -118,6 +121,7 @@ public static void testBaseBeforeClass() throws Exception {
     kafkaConsumer = new FakeKafkaConsumer().consume(vertx);
     kafkaConsumer.removeAllEvents();
     instanceReindex = new InstanceReindexFixture(client);
+    instanceIteration = new InstanceIterationFixture(client);
```

Comment:
Is this fixture needed by many test classes? If not, please consider defining it in the test classes that need it rather than here.

Reply:
Updated. Thanks